Src/Support/Google.Apis.Auth/OAuth2/ImpersonatedCredential.cs

```diff
@@ -35,65 +36,102 @@ namespace Google.Apis.Auth.OAuth2
     /// and https://cloud.google.com/iam/docs/impersonating-service-accounts
     /// for more information.
     /// </summary>
-    internal class ImpersonatedCredential : ServiceCredential, IOidcTokenProvider, IGoogleCredential, IBlobSigner
+    public sealed class ImpersonatedCredential : ServiceCredential, IOidcTokenProvider, IGoogleCredential, IBlobSigner
     {
         /// <summary>An initializer class for the impersonated credential. </summary>
-        new internal class Initializer : ServiceCredential.Initializer
+        new public sealed class Initializer : ServiceCredential.Initializer
         {
             /// <summary>
-            /// Gets or sets the source credential used to acquire the impersonated credentials.
+            /// Gets the service account to impersonate.
             /// </summary>
-            public GoogleCredential SourceCredential { get; set; }
+            public string TargetPrincipal { get; }
 
             /// <summary>
-            /// Gets or sets the impersonation options.
+            /// Gets the chained list of delegate service accounts. May be null or empty.
             /// </summary>
-            public ImpersonationOptions Options { get; set; }
+            public IEnumerable<string> DelegateAccounts { get; set; }
```

Comment:
Presumably it's okay to remove Options here, as it was previously an internal class.